pkgs/development/python-modules/openusd/default.nix

```
@@ -0,0 +1,133 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, pkgs
+, lib
+, cmake
+, doxygen
+, draco
+, graphviz-nox
+, ninja
+, setuptools
+, pyqt6
+, pyopengl
+, jinja2
+, pyside6
+, boost
+, numpy
+, git
+, tbb
+, opensubdiv
+, openimageio
+, opencolorio
+, osl
+, ptex
+, embree
+, openexr
+, flex
+, bison
+, qt6
+, python
+}:
+let
+  # Matches the pyside6-uic implementation
+  # https://code.qt.io/cgit/pyside/pyside-setup.git/tree/sources/pyside-tools/pyside_tool.py?id=e501cad66146a49c7a259579c7bb94bc93a67a08#n82
+  pyside-tools-uic = pkgs.writeShellScriptBin "pyside6-uic" ''
+    exec ${qt6.qtbase}/libexec/uic -g python "$@"
+  '';
+in
+buildPythonPackage rec {
+  pname = "OpenUSD";
+  version = "23.08";
+  src = fetchFromGitHub {
+    owner = "PixarAnimationStudios";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-3wM6stJnpznTU7Lb0vAdeO0/cr8n9rhvPZgT7PGAe04=";
```

I get a different hash? You could be building an old version in your cache.

```suggestion
    hash = "sha256-EDc+XQZeq5WbRKgzwMgD6Q7/wiL/Vc0ls2OKeOCjVQ8=";
```

---

My apologies, looks like I was indeed still building the last release.